src/doc/examples/query-objects.js

```diff
@@ -0,0 +1,74 @@
+import OLVectorLayer from 'ol/layer/Vector';
+import VectorSource from 'ol/source/Vector';
+import Feature from 'ol/Feature';
+import { Polygon } from 'ol/geom';
+import View from 'ol/View';
+import { Map, MapboxLayer, MapboxStyleLayer, VectorLayer } from '../../ol';
+import 'ol/ol.css';
+
+export default () => {
+  const addText = (text) => {
+    const element = document.getElementById('box');
+    element.innerHTML = text;
+  };
+
+  const map = new Map({
+    target: 'map',
+    view: new View({
+      center: [950690.34, 6003962.67],
+      zoom: 20,
+    }),
+  });
+
+  const mapboxLayer = new MapboxLayer({
+    url: `https://maps.geops.io/styles/base_bright_v1/style.json?key=${window.apiKey}`,
+  });
+
+  const poiLayer = new MapboxStyleLayer({
```

Comment:
Only a MapboxStyleLayer is queryable. So, I had to add a MapboxStyleLayer next to the MapboxLayer.